devai-cli/README.md

```diff
@@ -378,3 +378,215 @@ devai echo
 
 ### LangSmith LLM tracing configuration
 [Setup](../docs/tutorials/setup-langsmith.md) information.
+
+
+
+
+## Prompts Command
+
+The `prompts` command provides a powerful template system for managing and executing AI prompts. It allows you to create, manage, and reuse prompt templates for different types of code analysis, documentation, and review tasks. Templates are stored in YAML format and can be organized by categories, making them easy to find and maintain.
+
+### Basic Commands
+
+```bash
+# List all available prompts
+devai prompts list
+
+# Show details of a specific prompt
+devai prompts show security/web-security.yaml
+
+# Create a new prompt template
+devai prompts create
+
+# Initialize a prompts directory with sample templates
+devai prompts init
+
+# Configure prompts directory
+devai prompts config --set-path /path/to/prompts
```

Comment:
should this go before create command?

Trying `create` command first failed for me with:

`Error: Please set a custom prompts directory first using 'devai prompts config --set-path'`

Reply:
There's a set of defaults. The path is meant to be optional I'll investigate

Reply:
Updated order